### ISSUE TITLE
New release 0.4.1 not published  on npm

### ISSUE THREAD

**Issue by FireWizard52** [mikejac/node-red-contrib-google-smarthome#353]
Hello @Caprico85.

I saw you released a new version. However it has not been published to npm, yet.
Am I to curious, whether this will solve the local fulfillment issue? I do not expect, but who knows.

I discovered one typo in your new added documentation, so maybe you should correct that before publishing.
See: https://github.com/mikejac/node-red-contrib-google-smarthome/commit/31497943e97621fdb3b3d36acf5790e6ef3aea97

`If you experimented with local fulfillment and used "force local", remember to say "Hey Google, force **defaullt**" to
  re-enable the automatic fallback to cloud fulfillment.`

default instead of defaullt.

Regarding this line, I noted that if I use `force local` my device does not understand `force default`.
So I was not able to return to the default status. Don't know why.
First, after telling the device `force cloud`, it returned to a working state. After that I was able to set `force default`

If this had something to do with the language setting, I do not know. I use Dutch and English as languages.
Maybe worth to make a note on that.

Good job again!

Regards

**Comment by Caprico85** [comment]
Hi @FireWizard52,

> I saw you released a new version. However it has not been published to npm, yet.

It's published now. It took me quite a while to get the Github workflow for publishing running again. The "official" way of doing it always failed in the last weeks. Had to switch to another workflow action to get it to work.

> Am I to curious, whether this will solve the local fulfillment issue? I do not expect, but who knows.

Unfortunately no. Still doesn't work for me.

> I discovered one typo in your new added documentation, so maybe you should correct that before publishing.

 Thank you. Fixed it.

> Regarding this line, I noted that if I use force local my device does not understand force default.

I also have problems with `force default`. Usually I need several tries with different pronounciations of "default", until Google understands. Never had used `force cloud` though. I m using German as language. I'll do some testing and add a note to the readme.

**Comment by Paul-Reed** [comment]
I've been pretty short of time recently, and haven't been keeping up...
In fact,  I'm still using v0.3.14 which continues to deliver local fulfillment consistently well, so maybe it's a bug in the recent updates, and not a google server problem.

**Comment by FireWizard52** [comment]
Hi, @Caprico85 ,

Thank you for releasing a new version. I downloaded and installed it succesfully.
As you already said,  local fulfillment is not working with the new version 0.4.1.
As @Paul-Reed also said, I doubt as well, that the cause is Google. After @ckhmer1 's patch, it has worked for me a couple of days until it suddenly stopped. See: #345
As this is a new version with a new app.js version 2.6, I uploaded this version also to the Google Actions Console.
Strange fact is that I see in chrome://inspect and in the log in the Google Actions Console app.js version 2.5.
I do not see any error for each of my Google devices
Or am I to impatient?

Is there anything that we can check?

**Comment by FireWizard52** [comment]
Hi,

I wrote yesterday:

> Strange fact is that I see in chrome://inspect and in the log in the Google Actions Console app.js version 2.5.

Tonight, around 2 o'clock a.m., I noticed a first indication in the log of the Google Actions Console that app.js version 2.6 has been installed. So obvious I was too impatient. So it might take a couple of hours, before the new app.js has propagated to the various devices. I rebooted my Google devices and checked chrome://inspect.

Also there, I noted that the app.js version is 2.6.
I toke a similar screenshot as in #345.

![Screenshot_Chrome_Inspect5](https://user-images.githubusercontent.com/46283323/208461565-ddbc6e0f-f2d3-4cb7-a541-7ac489735661.png)

I cannot discover any significant difference between the one in #345 and this one. That could mean that the issue has not been resolved.

Any other opinion?
